debian/copyright

```diff
@@ -1,10 +1,10 @@
 Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: nicotine
-Upstream-Contact: Kip Warner <kip@thevertigo.com>
+Upstream-Contact: Nicotine+ Team <nicotine-team@lists.launchpad.net>
 Source: https://github.com/Nicotine-Plus/nicotine-plus/
 
 Files: *
-Copyright: Copyright (C) 2020 Nicotine+ Contributors
+Copyright: Copyright (C) 2020 Nicotine+ Team
```

Comment:
Copyright notice should be a range of Start-2021. Probably a good idea to update that everywhere.

Reply:
If this is a requirement in Debian copyright files, I'll change it here. My understanding is that specifying the start year alone is usually enough, and I'd like to avoid needless updates elsewhere.

Reply:
It's actually not a Debian specific requirement, but a legal requirement for all the source. A range is [needed](https://epiphany.law/articles/copyright/copyright-dates-single-year-or-range) too because there have been various releases over the years. Fortunately the following shell snippet will help you to update them all in a single command:

```bash
find -type f -name "*.py" -execdir sed -i 's/Copyright (C) 2020 Nicotine+ Contributors/Copyright (C) first_release-2021 Nicotine+ Team/g' {} \;
```